pylint/checkers/imports.py

```diff
@@ -431,6 +431,15 @@ def visit_if(self, node):
         if is_nested_allowed and \
                 any(node.nodes_of_class((astroid.Import, astroid.ImportFrom))):
             return
+        if isinstance(node, astroid.Assign):
```

Comment:
Can we rename the visit_if method to something else and point visit_if to it instead? Because it gets really confusing when looking at the code (inside a visit_if, but a check for astroid.Assign, which can trip the reviewer if he does not see the code from line 445.